src/screens/OrgList/OrgList.tsx

```diff
@@ -248,7 +259,10 @@ function OrgList(): JSX.Element {
                           orgLocation={datas.location}
                         />
                       );
-                    } else {
+                    } else if (
+                      data_2?.user.adminFor.length === 1 &&
+                      data_2?.user.adminFor[0]._id === datas._id
```

Comment:
The added logic seems to be getting too much to keep in the component markup. Can you abstract away the logic into a function that's called here.